lgc/include/lgc/state/PalMetadata.h

```diff
@@ -35,7 +35,9 @@
  */
 #pragma once
 
+#include "IntrinsDefs.h"
```

Comment:
Why do you need IntrinsDefs.h and PipelineState.h here?

Reply:
removed.